kratos/utilities/reduction_utilities.h

```diff
@@ -39,7 +66,7 @@ class SumReduction
     typedef TDataType   value_type;
     typedef TReturnType return_type;
 
-    TReturnType mValue = TReturnType(); // deliberately making the member value public, to allow one to change it as needed
+    TReturnType mValue = NullInitialized<TReturnType>::Get(); // deliberately making the member value public, to allow one to change it as needed
```

Comment:
i am really not convinced of this. Why would we need this public?

Reply:
I guess the intent was the option to specify an initial offset. I don't know if it's actually used anywhere so I didn't change it.